### ISSUE TITLE
iOS - Search - VoiceOver Does Not Announce Search Results after Searching

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#99]
While using VoiceOver, the user is not notified that any results were found after tapping the Search button.

When I searched for the word "test," it found ten screens that had it on.

**Comment by jfc3-dol** [comment]
VoiceOver still not announcing how many items it found and the focused at the top of the screen.

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we ensure that voiceover is reading the the number of search results found, as indicated in the text above the search box when searching a term?

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

VoiceOver is not reading the number of results that were found for a given term/word. VoiceOver reads "Search box" after tapping submit when I would expect to read '10 search results for "test"' when looking for information about "test".

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

I thought this issue had been fixed, but it's still an issue that I noticed when testing the application on iOS.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

VoiceOver is not reading the number of results that were found for a given term/word. VoiceOver reads "Search box" after tapping submit when I would expect VoiceOver to read '10 search results for "test"' when looking for information about "test".

**Comment by jfc3-dol** [comment]
@pbhatt17 

After searching, instead of reading the number of search results for a term, a person is looking for VocieOver is reading something like the following - "search submit search field test double-tap to edit use the rotor to access misspelled words." 

When searching for the word "test", I would expect VoiceOver to read '10 search results for "test"' instead of the above.

Plus, the VoiceOver focus ring shows up above the text '10 search results for "test"' in the shape of the search INPUT field. 

Please see the attached screenshot.

<img width="188" alt="ios - comply chain - search results - voiceover focus ring not over serach resutsl text as expected but shaped like INPUt field" src="https://user-images.githubusercontent.com/10423487/126560307-91b4a8c6-1a8b-443f-a02c-bbec8f35c761.png">

**Comment by jfc3-dol** [comment]
@pbhatt17

All the same things as last time are happening when you search with VoiceOver. The only difference is the look of the application is back to their previous shades of blue.

After searching, instead of reading the number of search results for a term, a person is looking for VocieOver is reading something like the following - "search submit search field test double-tap to edit use the rotor to access misspelled words."

When searching for the word "test", I would expect VoiceOver to read '10 search results for "test"' instead of the above.

Plus, the VoiceOver focus ring shows up above the text '10 search results for "test"' in the shape of the search INPUT field.

**Comment by jfc3-dol** [comment]
@pbhatt17 

After searching for "test", VoiceOver reads the text "Search" as "Search heading level one", when I would expect it to read the message '10 Search results for "Test"' so the VoiceOver user knows how many items were found for the term they looked for.

I did notice you moved the number of results message below the Search button, as we talked about.

**Comment by pbhatt17** [comment]
@jfc3-dol 
Please retest this issue. We pushed a fix for it.

**Comment by jfc3-dol** [comment]
@pbhatt17 

While using VoiceOver to search focuses on the search box and does not announce to the user how many results were found as expected. When I move to the search results message, it reads "two heading level two" and not the highlighted text. 

I searched for "test", so the search message is '10 search results for "test"'.
